src/NServiceBus.AcceptanceTests/DelayedDelivery/TimeoutManager/When_timeout_dispatch_fails.cs

```diff
@@ -23,14 +26,21 @@ await Scenario.Define<Context>()
                         .When((bus, c) =>
                         {
                             var options = new SendOptions();
-                            options.DelayDeliveryWith(TimeSpan.FromSeconds(1));
+                            options.DelayDeliveryWith(VeryLongTimeSpan);
                             options.RouteToThisEndpoint();
-                            return bus.Send(new MyMessage(), options);
+                            options.SetMessageId(c.TestRunId.ToString());
+
+                            return bus.Send(new MyMessage
+                            {
+                                Id = c.TestRunId
+                            }, options);
                         }))
-                .WithEndpoint<ErrorSpy>()
                 .Done(c => c.FailedTimeoutMovedToError)
                 .Repeat(r => r.For<AllTransportsWithoutNativeDeferral>())
-                .Should(c => Assert.AreEqual(5, c.NumTimesStorageCalled))
+                .Should(c =>
+                {
+                    Assert.IsFalse(c.DelayedMessageDeliveredToHandler, "Message was unexpectedly delivered to the handler");
```

Comment:
I think it doesn't hurt to repeat it, regardless of the done condition. There isn't a downside but it improves readability.